controller/argument_value_resolver.rst

```diff
@@ -146,10 +150,10 @@ and adding a priority.
 
         # app/config/services.yml
         services:
-            app.value_resolver.user:
-                class: AppBundle\ArgumentResolver\UserValueResolver
-                arguments:
-                    - '@security.token_storage'
+            # ...
+
+            AppBundle\ArgumentResolver\UserValueResolver:
+                # arguments is not needed if you have autowire above under _defaults
```

Comment:
A while ago I agreed with Ryan's idea ... but lately I'm aligning with Guilhem's idea.

Imagine that you land in this page and you see that message:

*"arguments is not needed if you have autowire above under _defaults"*

Well ... which "above" are we talking about? Which "_defaults" (whatever that is) are we talking about? etc.

This is truly critical because it's not only for newcomers ... but also for *"newcomers to the new DI features"* (I'm one of those 😊 ).

But of course Ryan is right and we can't add this continuously ... at least not by hand. Would some global include or custom directive help us automating this?